src/pdf-viewer.scss

```diff
@@ -0,0 +1,34 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-pdf-viewer;
+
+.#{$block} {
+  padding: 1rem;
+
+  &__dialog {
+    max-height: calc(100vh - 4rem);
```

Comment:
@stefanoScalzo The max height design spec you're referencing was fixed here.

Reply:
`the dialog height cannot go beyond 4 rem from the top and bottom of the screen."` Shouldn't it be `-4rem` per side, so `-8rem`?